Core/App/AppInstaller.php

```diff
@@ -128,7 +129,7 @@ private function createFolders()
     {
         // Check each needed folder to deploy
         foreach (['Plugins', 'Dinamic', 'MyFiles'] as $folder) {
-            if (!file_exists($folder) && !mkdir($folder)) {
+            if (!file_exists($folder) && !mkdir($folder) && !is_dir($folder)) {
```

Comment:
This was always a recommendation from PHP-CS do it in correct and more secure way.